quantmsio/core/common.py

```
@@ -0,0 +1,128 @@
+}
+MAXQUANT_PSM_MAP = {
+    "Sequence": "sequence",
+    "Proteins": "mp_accessions",
+    "PEP": "posterior_error_probability",
+    "Modified sequence": "peptidoform",
+    "Reverse": "is_decoy",
+    "m/z": "observed_mz",
+    "Scan number": "scan",
+    "Retention time": "rt",
+    "Charge": "precursor_charge",
+    "Modified sequence": "peptidoform",
```

Ruff

Dictionary key literal `"Modified sequence"` repeated

Remove repeated key literal `"Modified sequence"`

(F601)